### PR TITLE
feat(tiflow): add periodics pipeline for cdc kafka test

### PR DESCRIPTION
**Pull request by purelind**
Add periodics pipeline for cdc kafka test

### CODE REVIEW

**Comment by purelind** [comment]
/hold

need to check if prow periodics type support agent jenkins

File: pipelines/pingcap/tiflow/latest/periodics_cdc_integration_kafka_test.groovy [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                        def pdBranch = "master"
+                        def tikvBranch = "master"
+                        def tiflashBranch = "master"
+                        retry(2) {
+                            sh label: "download third_party", script: """
+                                export TIDB_BRANCH=${tidbBranch}
+                                export PD_BRANCH=${pdBranch}
+                                export TIKV_BRANCH=${tikvBranch}
+                                export TIFLASH_BRANCH=${tiflashBranch}
+                                cd ../tiflow && ./scripts/download-integration-test-binaries.sh master && ls -alh ./bin
```
Comment (wuhuizuo) [inline pipelines/pingcap/tiflow/latest/periodics_cdc_integration_kafka_test.groovy:71]: Is this script in tiflow repo?

Comment (wuhuizuo) [reply pipelines/pingcap/tiflow/latest/periodics_cdc_integration_kafka_test.groovy:71]: `third_party_download/tiflow`?
but the tiflow codes are checked out in `${WORKSPACE}/tiflow` path.

**Review by wuhuizuo — APPROVED** [review]
Let's try it firstly, then fix the issues one by one.

File: pipelines/pingcap/tiflow/latest/periodics_cdc_integration_kafka_test.groovy (hunk 1 above)
Comment (purelind) [reply pipelines/pingcap/tiflow/latest/periodics_cdc_integration_kafka_test.groovy:71]: yes

**Pull request merged**